sqls/test.sql

```diff
@@ -2,6 +2,6 @@ SELECT
     a,
     b,
     e,
-    c + d
+    c + d AS result
```

Comment:
🚫 **[sqlfluff]** <[RF04](https://docs.sqlfluff.com/en/stable/rules.html#rule-RF04)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[references.keywords] - Keywords should not be used as identifiers.
<!-- __reviewdog__:ChBhMWRmYTQyMjA4YzI5MGZiEghzcWxmbHVmZg== -->